tests/test_etcd3.py

```diff
@@ -88,6 +89,90 @@ def test_delete_key(self, etcd):
         with pytest.raises(etcd3.exceptions.KeyNotFoundError):
             etcd.get('/doot/delete_this')
 
+    def test_watch_key(slef, etcd):
+        def update_etcd(v):
+            etcdctl('put', '/doot/watch', v)
+            out = etcdctl('get', '/doot/watch')
+            assert base64.b64decode(out['kvs'][0]['value']).decode('utf-8') == v
```

Comment:
All the `decode('utf-8')` calls should probably be rewritten to compare against `utils.to_bytes`